cmake/dependencies/toml11.cmake

```diff
@@ -64,10 +64,10 @@ set(openPMD_toml11_src ""
     "Local path to toml11 source directory (preferred if set)")
 
 # tarball fetcher
-set(openPMD_toml11_tar "https://github.com/ToruNiina/toml11/archive/refs/tags/v4.2.0.tar.gz"
```

Comment:
This was out-of-sync, we use `3.7.1` also in `openPMD_toml11_branch` and documentation.